### PR TITLE
Create Copyable component

### PR DESCRIPTION
**Pull request by rasmusguterstam**
## What?
- Add `Copyable` component to `hedvig-ui`
- Add this bad boy to Claim: Member Information as requested

## Why?
- For some reason, IEX could not copy the link 

## Optional screenshots

https://user-images.githubusercontent.com/51323202/129224431-9ca6793d-e309-446f-b958-1467e3a814a1.mov

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [x] Codescouted
- [ ] Unit tests written
- [x] Tested locally

### CODE REVIEW

File: shared/hedvig-ui/copyable.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+export const Copyable: React.FC<{
+  children: React.ReactNode
+  onClick: () => void
+  copyLabel?: PopoverLabel
+}> = ({ copyLabel, children, onClick }) => {
+  const [copied, setCopied] = useState(false)
+
+  const { before = 'Copy', after = 'Copied!' } = copyLabel ?? {}
+
```
Comment (joacimastrom) [inline shared/hedvig-ui/copyable.tsx:56]: Maybe reset copied state after a short delay? :)
```suggestion
useEffect(() => setTimeout(() => setCopied(false), 4000), [copied])
```

Comment (joacimastrom) [reply shared/hedvig-ui/copyable.tsx:56]: useEffect also needs to be imported on top

File: shared/hedvig-ui/popover.tsx [hunk 2]
```diff
@@ -66,16 +66,8 @@ export const Popover: React.FC<PopoverProps> = ({
...
-        if (onOpen) {
-          onOpen()
-        }
-      }}
-      onMouseLeave={() => {
-        if (onClose) {
-          onClose()
-        }
-      }}
+      onMouseOver={() => onOpen?.()}
```
Comment (joacimastrom) [inline shared/hedvig-ui/popover.tsx:69]: Wow, didn't know you could do this for function calls

**Comment by joacimastrom** [comment]
While the animation looks very nice (👏 ) I think the UX might be more clear and intuitive if we just always show the copy-icon?  Especially since the link and the icon does different things, would be nice to be able to copy without having to hover first 🤷‍♂️

File: shared/hedvig-ui/copyable.tsx (hunk 1 above)
Comment (rasmusguterstam) [reply shared/hedvig-ui/copyable.tsx:56]: Oo that's really neat!

**Review by joacimastrom — APPROVED** [review]

**Pull request merged**